sinopia_client/src/api/LDPApi.js

```diff
@@ -575,6 +575,51 @@
         });
     }
 
+    /**
+     * Get metadata for a given resource template.
+     * Get serialization of JSON for a given NON-RDF resource.
+     * @param {String} groupID The group who is defining it&#39;s own resources or graph within Sinopia.
+     * @param {String} resourceID The UUID for the resource defined and managed within Sinopia.
+     * @return {Promise} a {@link https://www.promisejs.org/|Promise}, with an object containing data of type {@link module:model/Resource} and HTTP response
+     */
+    this.getNonRdfResourceWithHttpInfo = function(groupID, resourceID) {
+      var postBody = null;
+
+      // verify the required parameter 'groupID' is set
+      if (groupID === undefined || groupID === null) {
+        throw new Error("Missing the required parameter 'groupID' when calling getResource");
+      }
+
+      // verify the required parameter 'resourceID' is set
+      if (resourceID === undefined || resourceID === null) {
+        throw new Error("Missing the required parameter 'resourceID' when calling getResource");
+      }
+
+
+      var pathParams = {
+        'groupID': groupID,
+        'resourceID': resourceID
+      };
+      var queryParams = {
+      };
+      var collectionQueryParams = {
+      };
+      var headerParams = {
+      };
+      var formParams = {
+      };
+
+      var authNames = ['RemoteUser'];
+      var contentTypes = ['application/json'];
+      var accepts = ['application/json'];
+      var returnType = Resource;
+
+      return this.apiClient.callApi(
+        '/repository/{groupID}/{resourceID}', 'GET',
+        pathParams, queryParams, collectionQueryParams, headerParams, formParams, postBody,
+        authNames, contentTypes, accepts, returnType
+      );
+    }
```

Comment:
my hope is that the need for this new method will be obviated by the work in PR #85, and that instead callers can use something like the following to get back non-RDF from trellis:
```javascript
instance.getResourceWithHttpInfo(groupId, slug, { acceptEncoding: 'application/json' })
```
(see https://github.com/LD4P/sinopia_server/pull/85/files#diff-9c98b1f96912899c02b40c5bb91a07dcR214)

Reply:
`s/acceptEncoding/accept` here I think?

Reply:
I removed the added method. The work you did on #85 worked just fine.